plugins/nvme/fu-nvme-device.c

```diff
@@ -0,0 +1,330 @@
+/*
+ * Copyright (C) 2018 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include <fcntl.h>
+#include <string.h>
+#include <sys/errno.h>
+#include <sys/ioctl.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+
+#include <glib/gstdio.h>
+
+#include <linux/nvme_ioctl.h>
```

Comment:
So this adds an implicit dependency of kernel [4.4 right](https://github.com/torvalds/linux/commit/9d99a8dda154f38307d43d9c9aa504bd3703d596)?  Unless you end up wanting to run this plugin on RHEL 7.x it would be good to check at plugin startup for at least this kernel version and fail otherwise.

Reply:
Ahh, I should check for this in meson.build rather relying on a specific kernel version -- good catch, thanks.

Reply:
Fixed.